commons/src/main/java/gov/cms/qpp/conversion/util/MeasuredInputStreamSupplier.java

```diff
@@ -40,6 +28,18 @@ private MeasuredInputStreamSupplier(InputStream source) {
 		this.size = byteArray.length;
 	}
 
+	/**
+	 * Terminally uses an {@link InputStream} to create a {@link MeasuredInputStreamSupplier}
+	 *
+	 * @param source the source for the new {@link MeasuredInputStreamSupplier}. Must not be null.
+	 * @return a new {@link MeasuredInputStreamSupplier} from the given {@link InputStream}
+	 */
+	public static MeasuredInputStreamSupplier terminallyTransformInputStream(InputStream source) {
```

Comment:
I feel that a `public static` constructor should be above a `private` constructor.  This may have to do with https://jira.cms.gov/browse/QPPCT-362

Reply:
k